selfdrive/car/honda/hondacan.py

```diff
@@ -58,27 +58,35 @@ def create_acc_commands(packer, enabled, active, accel, gas, stopping, car_finge
   standstill = 1 if active and stopping else 0
   standstill_release = 1 if active and not stopping else 0
 
-  acc_control_values = {
-    # setting CONTROL_ON causes car to set POWERTRAIN_DATA->ACC_STATUS = 1
-    "CONTROL_ON": control_on,
-    "GAS_COMMAND": gas_command,  # used for gas
-    "ACCEL_COMMAND": accel_command,  # used for brakes
-    "BRAKE_LIGHTS": braking,
-    "BRAKE_REQUEST": braking,
-    "STANDSTILL": standstill,
-    "STANDSTILL_RELEASE": standstill_release,
-  }
-  commands.append(packer.make_can_msg("ACC_CONTROL", bus, acc_control_values))
-
-  acc_control_on_values = {
-    "SET_TO_3": 0x03,
-    "CONTROL_ON": enabled,
-    "SET_TO_FF": 0xff,
-    "SET_TO_75": 0x75,
-    "SET_TO_30": 0x30,
-  }
-  commands.append(packer.make_can_msg("ACC_CONTROL_ON", bus, acc_control_on_values))
-
+  if car_fingerprint in HONDA_BOSCH_RADARLESS:
+    control_on = 1 if enabled else 0
+    acc_control_values = {
+      "CONTROL_ON": control_on,
+      "CONTROL_OFF": 1,  # TODO: not sure this signal is needed
+      "ACCEL_COMMAND": accel_command,
+      "STANDSTILL": standstill,
```

Comment:
the signals for this message name defined already below, are they common at all in the new acc control msg? are there weird constant setmes?

Reply:
Some are common, but might have different values like CONTROL_ON. I moved common messages out of the if statement.